openedx/core/djangoapps/user_api/urls.py

```diff
@@ -4,7 +4,7 @@
 
 from .accounts.views import AccountView
 from .preferences.views import PreferencesView, PreferencesDetailView
-
+from ..profile_images.views import ProfileImageView
 from django.conf.urls import patterns, url
```

Comment:
group imports:
* standard library
* external packages (incl third party)
* current package.

alphabetize based on the first non-keyword (e.g. whatever occurs after `import` or `from`)

in other repos we are avoiding relative imports.